.github/workflows/build_wheel_windows_arm64.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Windows Arm64 Torchvision Wheel
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+    tags:
+        # NOTE: Binary build pipelines should only get triggered on release candidate builds
+        # Release candidate tags look like: v1.11.0-rc1
+        - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+  workflow_dispatch:
+    inputs:
+      python_version:
+        description: "Python version"
+        required: true
+        type: choice
+        options:
+          - Python312
+          - Python311
+        default: Python312
+      msvc_version:
+        description: "MSVC Version"
+        required: true
+        type: choice
+        options:
+          - 'latest'
+          - '14.40'
+          - '14.36'
+        default: 'latest'
+      build_type:
+        description: "Build type"
+        required: true
+        type: choice
+        options:
+          - Release
+          # - Debug (not supported for wheel)
+        default: Release
+      repository_name:
+        description: "torchvision repository (username/repo)"
+        required: true
+        type: string
+        default: "pytorch/vision"
+      repository_branch:
+        description: "repository branch"
+        required: true
+        type: string
+        default: "main"
+      build_version:
+        description: "Optional: build version for wheel file name (e.g. 2.6.0); leave empty for default naming"
+        required: false
+        type: string
+        default: ""
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  trigger-test-infra:
+    uses: alinpahontu2912/test-infra/.github/workflows/build_wheels_windows_arm64.yml@winarm64_wheels
```

Comment:
> [!WARNING]
> This will not work, **you need to merge whatever changes are in this fork into upstream _pytorch/test-infra_** or else you'll continually run into issues with trying to have Github schedule these jobs with **"Called workflows cannot be queued onto self-hosted runners across organizations/enterprises. Failed to queue this job. Labels: 'windows-11-arm64'."**